tools/schemapi/schemapi.py

```diff
@@ -562,11 +562,17 @@ def to_dict(self, validate=True, ignore=None, context=None):
             try:
                 self.validate(result)
             except jsonschema.ValidationError as err:
-                raise SchemaValidationError(self, err)
+                raise SchemaValidationError(self, err) from err
```

Comment:
```suggestion
                # We do not raise `from err` and therfore ignore 
                # the B904 linting rule as else the resulting
                # traceback is very long as it contains part
                # of the Vega-Lite schema. It would also first
                # show the less helpful ValidationError instead of
                # the more user friendly SchemaValidationError
                raise SchemaValidationError(self, err)  # noqa: B904
```

Reply:
Same change as above, `raise from None` instead of `rase # noqa: B904`